docs/src/visualization.md

```diff
@@ -393,14 +393,23 @@ julia> fig, axes = plot(sol)
 julia> axes[1,1].title = "New title for subplot (1,1)"
 ```
 
-Trixi also supports interactive surface plots using `iplot`:
+Trixi also supports interactive surface plots using `iplot`.
+After executing `examples/unstructured_2d_dgsem/elixir_euler_wall_bc.jl`, we can run
 ```julia
 julia> iplot(sol)
 ```
 This will open up an interactive visualization window:
 
-![makie-example](https://user-images.githubusercontent.com/1156048/127914785-bb87cff5-92b2-4e72-9ed1-9c04f4f81a73.png)
+![makie-example](https://user-images.githubusercontent.com/1156048/131539850-affd6a48-f152-4d67-8ed1-cac939daaf75.png)
```

Comment:
If I am not mistaken, this plot still uses the `:viridis` color map. Following POLA (principle of least astonisment), shouldn't this be generated using the new default color map `:inferno` as well?

Reply:
At least the color bar next to the plot is definitely sickly green; the plot itself could already be `:inferno`.

Reply:
Oops - left a hanging `:viridis`. Fixed - regenerating plots now

Reply:
👍